crates/sui-framework/packages/sui-system/sources/validator_set.move

```diff
@@ -556,6 +557,21 @@ module sui_system::validator_set {
         &self.staking_pool_mappings
     }
 
+    public(friend) fun pool_exchange_rates(
```

Comment:
Do we have tests for this function? If not, how critical is it?

Reply:
Added a test.